src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -311,6 +311,11 @@ public Integer getTtl() {
         return environmentProperties;
     }
 
+    @Restricted(NoExternalUse.class)
+    public boolean getCustomDomain() {
```

Comment:
Wouldn't this make more sense named as `isCustomDomain()`?

Reply:
Just trying to make a least intrusive change to make this plugin work with [Configuration as code](https://github.com/jenkinsci/configuration-as-code-plugin)